app/controllers/users_controller.rb

```diff
@@ -43,4 +46,29 @@ def set_user
   def user_params
     params.require(:user).permit(:full_name, :email, :role, :password, :password_confirmation, :current_password, registers: [])
   end
+
+end
+
+class UsersController::InvitationsController < Devise::InvitationsController
+
+  def create
+    flash[:notice] = 'Invite sent to ' + resource_params[:email]
+    redirect_to '/team'
```

Comment:
I've actually removed this overriding method altogether since, it was part of testing something